src/microbe_stage/Microbe.cs

```diff
@@ -1427,22 +1427,39 @@ private void OnIGotAddedToColony()
             return;
 
         var oldRotation = Rotation;
-        var vectorToParent = GlobalTransform.origin - ColonyParent.GlobalTransform.origin;
-        ChangeNodeParent(ColonyParent);
 
-        var vectorToParentRotated = vectorToParent.Rotated(Vector3.Down, Rotation.y);
-        var vectorToMembrane = Membrane.GetExternalOrganelle(vectorToParentRotated.x, vectorToParentRotated.y);
+        // Gets the global rotation of the parent
+        var globalParentRotation = ColonyParent.GlobalTransform.basis.GetEuler();
+
+        // A vector from the parent to me
+        var vectorFromParent = GlobalTransform.origin - ColonyParent.GlobalTransform.origin;
+
+        // This vector gets as if I had no rotation.
+        // This is important, because GetExternalOrganelle only works with non-rotated microbes
+        var vectorToParentRotated = vectorFromParent.Rotated(Vector3.Up, Rotation.y);
```

Comment:
If I rotate "vectorFromParent", it becomes "vectorToParentRotated"? Is this a typo, or a logical error?